### ISSUE TITLE
eCP chart fails in Build 4007 (Oct 07 2021)

### ISSUE THREAD

**Issue by v1sionmaster** [GoldenCheetah/GoldenCheetah#4047]
Build 4007 (Oct 07 2021)
The eCP Trend fails to display the trend and shows the following error in the console. 
![Screenshot 2021-10-07 115850](https://user-images.githubusercontent.com/38918646/136371575-5c9e8b27-9c6a-4d23-b752-6df62cf70f72.jpg)

Python Console (3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:58:18) [MSC v.1900 64 bit (AMD64)])
»> D:\Golden Cheetah 3_6\Golden Cheetah\lib\site-packages\numpy\_distributor_init.py:32: UserWarning: loaded more than 1 DLL from .libs:
D:\Golden Cheetah 3_6\Golden Cheetah\lib\site-packages\numpy\.libs\libopenblas.PYQHXLWQ7VESDPUVUADXEVJOB GHJPAY.gfortran-win_amd64.dll
D:\Golden Cheetah 3_6\Golden Cheetah\lib\site-packages\numpy\.libs\libopenblas.XWYDX2IKJW2NMTWSFYNGFUWKQU 3LYTCZ.gfortran-win_amd64.dll stacklevel=l)
Traceback (most recent call last):
  File "D:\Golden Cheetah 3_6\Golden Cheetah\lib\site-packages\asteval\_version.py", line 7, in <module> from importlib.metadata import version, PackageNotFoundError
ModuleNotFoundError: No module named 1importlib.metadata'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "<string>", line 10, in <module>
  File "D:\Golden Cheetah 3_6\Golden Cheetah\lib\site-
packages\lmfit\_init_.py"/ line 36, in <module>
    from asteval import Interpreter File "D:\Golden Cheetah 3_6\Golden Cheetah\lib\site-
packages\asteval\_init_.py", line 27, in <module>
    from ._version import _version_
  File "D:\Golden Cheetah 3_6\Golden Cheetah\lib\site-packages\asteval\_version.py", line 11, in <module> from importlib_metadata import version, PackageNotFoundError
ModuleNotFoundError: No module named 'importlib_metadata'

Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers
